Makefile

```diff
@@ -105,7 +105,7 @@ generate-code: ## Regenerate clientset, deepcopy funcs, listers and informers
 
 .PHONY: generate-crds
 generate-crds: ## Regenerate CRDs in the Helm chart and examples
-	controller-gen crd:maxDescLen=0 paths=./pkg/... output:crd:artifacts:config=./deploy/
+	bin/controller-gen crd:maxDescLen=0 paths=./pkg/... output:crd:artifacts:config=./deploy/
```

Comment:
Add bin folder to path instead

Reply:
It was already exported int PATH, reverted this change.